cpg-language-python/src/test/kotlin/de/fraunhofer/aisec/cpg/frontends/python/PythonFrontendTest.kt

```diff
@@ -1586,6 +1586,29 @@ class PythonFrontendTest : BaseTest() {
         }
     }
 
+    @Test
+    fun test() {
+        val directoryPath = Path.of("/home", "lshala", "repos", "nova")
+        val excludedDirs = listOf("tests", "accelerator")
+        val tr =
+            analyze(".py", directoryPath, usePasses = true) {
+                it.registerLanguage<PythonLanguage>()
+                it.excludedDirs(excludedDirs)
```

Comment:
@oxisto As discussed it would be helpful to have further exclusion patterns. Would it be useful to have both a straightforward exclusion list like `excludedDirs` and an additional property (`exlusionPatterns`?) for using regex patterns, e.g.
```
val exclusionPatterns = listOf(
   Regex(".*\\.test\\..*")
   Regex(".*__tests__.*")
)
```
(Test test() will be removed)

Reply:
Yes it would be helpful :)

Reply:
instead of the first proposed way of filtering via two properties, I would use only one property that allows filtering via strings or regex